cypress/support/index.js

```diff
@@ -8,8 +8,10 @@ Cypress.Screenshot.defaults({
 
 Cypress.on(`window:before:load`, win => {
   cy.stub(win.console, `error`).callsFake(msg => {
-    cy.now(`task`, `error`, msg);
-    throw new Error(msg);
+    if (!win.location.href.startsWith('https://www.bbc.com/usingthebbc')) {
+      cy.now(`task`, `error`, msg);
+      throw new Error(msg);
+    }
   });
 });
```

Comment:
Consider the approach below to track problematic URLs.

```
const KNOWN_URLS = ['https://www.bbc.com/usingthebbc'];

const isKnownUrl = testLocation =>
  KNOWN_URLS.some(url => testLocation.startsWith(url));

Cypress.on(`window:before:load`, win => {
  cy.stub(win.console, `error`).callsFake(msg => {
    if (!isKnownUrl(win.location.href)) {
      cy.now(`task`, `error`, msg);
      throw new Error(msg);
    }
  });
});
```

Reply:
Great suggestion - means the tests will fail if we encounter problems on other URLs that we're not expecting.